src/splitAt.js

```diff
@@ -0,0 +1,21 @@
+var _curry2 = require('./internal/_curry2');
+
+
+/**
+ * Splits a given list or string at a given index.
+ *
+ * @func
+ * @memberOf R
+ * @category List
+ * @sig Number -> [a] -> [[a], [a]]
+ * @param {Number} a
+ * @param {Array|String} b
+ * @return {Array}
+ * @example
+ *
+ *      R.splitAt(1, [1, 2, 3]);          //=> [[1], [2, 3]]
+ *      R.splitAt(5, 'hello world');      //=> ['hello', ' world']
+ */
+module.exports = _curry2(function splitAt(index, array) {
+  return [array.slice(0, index), array.slice(index)];
```

Comment:
We should use `R.slice` here to support array-like objects which don't provide a `slice` method.